images/icons/close.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" style="-ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);"><path d="M8 19a3 3 0 0 1-3-3V8a3 3 0 0 1 3-3a1 1 0 0 0 0-2a5 5 0 0 0-5 5v8a5 5 0 0 0 5 5a1 1 0 0 0 0-2zm7.71-3.29a1 1 0 0 0 0-1.42L13.41 12l2.3-2.29a1 1 0 0 0-1.42-1.42L12 10.59l-2.29-2.3a1 1 0 0 0-1.42 1.42l2.3 2.29l-2.3 2.29a1 1 0 0 0 0 1.42a1 1 0 0 0 1.42 0l2.29-2.3l2.29 2.3a1 1 0 0 0 1.42 0zM16 3a1 1 0 0 0 0 2a3 3 0 0 1 3 3v8a3 3 0 0 1-3 3a1 1 0 0 0 0 2a5 5 0 0 0 5-5V8a5 5 0 0 0-5-5z" fill="#fff"/><rect x="0" y="0" width="24" height="24" fill="rgba(0, 0, 0, 0)" /></svg>
```

Comment:
Everything seems great, though I'm not quite sure about the close icon 😐

Reply:
I couldn't find a better icon in the iconset.

Reply:
@GrahamSH-LLK I understand that, but could I edit the icon?

Reply:
Not easily...

Reply:
@GrahamSH-LLK I edited it, and here is the code:
`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0,0,24,24"><g transform="translate(-228,-168)"><g data-paper-data="{&quot;isPaintingLayer&quot;:true}" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" style="mix-blend-mode: normal"><path d="M243,184.0058c-0.26664,0 -0.52223,-0.10649 -0.71,-0.2958l-2.29,-2.3l-2.29,2.3c-0.18777,0.18931 -0.44336,0.2958 -0.71,0.2958c-0.26664,0 -0.52223,-0.10649 -0.71,-0.2958c-0.18931,-0.18777 -0.2958,-0.44336 -0.2958,-0.71c0,-0.26664 0.10649,-0.52223 0.2958,-0.71l2.3,-2.29l-2.3,-2.29c-0.39212,-0.39212 -0.39212,-1.02788 0,-1.42c0.39212,-0.39212 1.02788,-0.39212 1.42,0l2.29,2.3l2.29,-2.3c0.39212,-0.39212 1.02788,-0.39212 1.42,0c0.39212,0.39212 0.39212,1.02788 0,1.42l-2.3,2.29l2.3,2.29c0.18931,0.18777 0.2958,0.44336 0.2958,0.71c0,0.26664 -0.10649,0.52223 -0.2958,0.71c-0.18777,0.18931 -0.44336,0.2958 -0.71,0.2958z" fill="#ffffff"/><path d="M228,192v-24h24v24z" fill-opacity="0" fill="#000000"/></g></g></svg>`